ntbs-service/Services/AdImportService.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using System.DirectoryServices;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using ntbs_service.DataAccess;
+using ntbs_service.Models.Entities;
+
+namespace ntbs_service.Services
+{
+
+    public interface IAdImportService
+    {
+        Task RunCaseManagerImport();
+    }
+    public class AdImportService : IAdImportService
+    {
+        private readonly IAdDirectoryFactory _adDirectoryFactory;
+        private readonly IReferenceDataRepository _referenceDataRepository;
+        private readonly IUserRepository _userRepository;
+
+        private static readonly Regex DistinguishedNameRegex = new Regex("(CN=)(Global.NIS.NTBS[^,]+)(,.*)");
+
+        public AdImportService(IAdDirectoryFactory adDirectoryFactory, IReferenceDataRepository referenceDataRepository, IUserRepository userRepository)
+        {
+            _adDirectoryFactory = adDirectoryFactory;
+            _referenceDataRepository = referenceDataRepository;
+            _userRepository = userRepository;
+        }
+
+        // get those values from settings

```

Comment:
Which values is this comment trying to describe?

Reply:
Oops I think that was leftover from some old method that I annotated for my own benefit, can be removed now